index.html

```diff
@@ -5451,6 +5451,17 @@ <h3>Base Context</h3>
 This section lists cryptographic hash values that might change during the
 Candidate Recommendation phase based on implementer feedback that requires
 the referenced files to be modified.
+<br><br>
+The Working Group is expecting all of the terms and URLs supplied in the
+JSON-LD Context to be either stabilized, or removed, before the publication of
+this specification as a Proposed Recommendation. While that means that this
+specification could be delayed if dependencies such as [[?VC-DATA-INTEGRITY]],
+[[?VC-JOSE-COSE]], SD-JWT, [[?VC-JSON-SCHEMA-2023]], or status list
```

Comment:
```suggestion
[[?VC-JOSE-COSE]], [[?VC-JSON-SCHEMA-2023]], or status list
```

At this moment at least, SD-JWT is not even on the list of the [official publications of the WG](https://www.w3.org/groups/wg/vc/publications/), i.e., talking about the PR phase of that document is meaningless.

Reply:
PR #1242 adds SD-JWT terms to a normative document (the base context) that the group is publishing. It highlights the problem that @jyasskin raised... we now have a normative dependency on a document at IETF that is almost certainly not going to be finished on the timeline that the VCWG is on. While it's not on the official publication of the WG, the VC-JOSE-COSE specification, and the base context have a normative dependency on it.

Again, the base context having a dependency on it isn't a big deal... by the time we hit REC (Summer 2024), SD-JWT should be stable enough (in the properties that it uses)... and if it's not, the group can remove references to it from the base context (which is what this PR says for those properties, and any other unstable properties).

Reply:
Ok. 

B.t.w., let us be precise. By believe that in this context (sic!) our dependency is _on the URL-s_ and not on the details of the respective specifications. To take an example, we have a dependency on `http://schema.org/name` but not on the content of the HTML page that describes that particular term. Maybe it is worth emphasizing this, because the text reads as we had a normative dependency on the respective specification as a whole. Is that correct?

And yes, the SD-JWT specification might be in flux, but the URLs we use may not...

Reply:
Yes, correct, our dependency is on the URLs, primarily. 

@jyasskin's point was that we could accidentally create a bi-directional dependency. Not only do we lock in the terms and URLs from our standpoint, but we could make it such that the spec that we reference then couldn't change the terms/URLs if they wanted to (because we'd ship something that they weren't ready to ship). 

For example, if SD-JWT decides to change the `...` term to `o_O`, because we VCWG shipped with `...`, that might cause them to need to support both properties in their spec for backwards compatibility reasons, which is something they wouldn't do if we didn't include those properties in our base context.

The mitigation to all of that is just keeping all of the groups in sync w/ one another and being ready to pull the terms that are not stable enough as we go into Proposed Rec. We have 9-ish more months for all of this to stabilize, so making the decision now is premature... we'll reserve the right to make the decision later (which is what this PR attempts to say).

Reply:
yes, the v2 context has terms from several related documents, if any of them stall / fail or are blocked indefinitely, the bundling strategy could bite us, the list includes:

- vc-data-integrity
- vc-status-list
- vc-json-schema
- sd-jwt (IETF)

Possible there are other documents that could cause problems beyond these.

vc-jose-cose does not define any terms, hence it has nothing appear in the v2 context.

Reply:
> vc-jose-cose does not define any terms, hence it has nothing appear in the v2 context, but normative dependency issues would still exist.

Debatable... One of our specs should speak to the fact of all the ".../jose#" and ".../jwt#" entries in the core context now... same for SD-JWT properties. If the JOSE group decides they don't want to use those URLs, or SD-JWT changes (or vice versa), it creates the same issue... and the resolution is the same as all the other specs -- features will be cut/removed if they get in the way for REC.